test/unique-selector.js

```diff
@@ -35,6 +35,15 @@ describe( 'Unique Selector Tests', () =>
     expect( uniqueSelector ).to.equal( 'body > :nth-child(1)' );
   } );
 
+  it( 'Classes', () =>
```

Comment:
Right, I see, there are not more descriptive indeed. Sorry, I cant remember the codebase, I havent work on the project the last months.